langchain/tools/zapier/tool.py

```diff
@@ -1,6 +1,6 @@
 """## Zapier Natural Language Actions API
 \
-Full docs here: https://nla.zapier.com/api/v1/dynamic/docs
+Full docs here: https://nla.zapier.com/api/v1/docs
```

Comment:
This and the other changes for docs should remain `dynamic/docs`.  It's a bit of an inconsistency on our end when it comes to expressing which route to use.